<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>552</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>JetBrains Mono Light</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_confirm">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>241</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QSlider" name="slider_kernel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Kernel size</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbb_filter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>選擇filter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gausian filter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Median filter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bilateral filter</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Filter Type</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_kernel_size">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>47</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>size</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_gray">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Grayscale</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>531</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>Preview</string>
    </property>
    <widget class="QLabel" name="lb_show">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>491</width>
       <height>331</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_reset">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>340</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Addtional</string>
    </property>
    <widget class="QSlider" name="slider_color">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="slider_space">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>sigmaColor </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>sigmaSpace </string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>btn_confirm</zorder>
   <zorder>groupBox_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>844</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
